src/box/box.cc

```diff
@@ -790,6 +804,141 @@ wal_stream_apply_dml_row(struct wal_stream *stream, struct xrow_header *row)
 	return -1;
 }
 
+/* Storage, adding rows to a rlist to separate mixed transactions. */
+struct wal_row {
+	/* Base class. */
+	struct xrow_header row;
+	/* List of rows to stacked in the nodes_rows array. */
+	struct rlist in_row_list;
+	/* A growing identifier to track lsregion allocations. */
+	int64_t lsr_id;
+};
+
+/*
+ * Callback to stash row and row bodies upon receipt, used to recover
+ * mixed transactions.
+ */
+static struct wal_row *
+wal_stream_save_row(struct wal_stream *stream, struct xrow_header *row)
+{
+	static int64_t lsr_id = 0;
+	struct lsregion *lsr = &stream->lsr;
+	struct wal_row *new_row = (struct wal_row *)xlsregion_alloc(lsr,
+				   sizeof(struct wal_row), ++lsr_id);
```

Comment:
??

Reply:
Your argument alignment is wrong, please fix it like in my example.
Checkpatch doesn't catch this because it sees an opening bracket of `(struct wal_row *)` above, I suppose.